lab-kyle/test/main-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var main = require(__dirname + '/../main');
+var equal = require('assert').equal;
+
+describe('testing module main', function(){
+  describe('testing main function', function(){
+    it('with process.argv[2] = kyle should return hello kyle', function(){
```

Comment:
You are not actualy testing process.argv in this file ... you will want to make sure that you set process.argv[2] in order to test it :)